### PR TITLE
Implement unit tests with Jest

### LINKED ISSUE #18: Implement test suite
We plan to implement automated tests with [jest](https://facebook.github.io/jest/), and both cli and renderer should be covered.

### PR DESCRIPTION
**Pull request by mRcfps**
Closes #18.

However there are some places where our tests currently fail to cover:

- Prompts of `tuture init` and `tuture destroy`.
- Whether `tuture-renderer` is called and browser is opened automatically.

### CODE REVIEW

File: tests/tuture.test.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  });
+
+  describe('unknown args', () => {
+    it('should exit(1) when unknown commands are given', () => {
+      expect(run(['foobar']).status).toBe(1);
+    });
+
+    it('should exit(1) when unknown options are given', () => {
+      expect(run(['--foobar']).status).toBe(1);
+    });
```
Comment (pftom) [inline tests/tuture.test.js]: May `unix` command contains `-args`, like `tuture -f`. Should we test it ?

```javascript
  it('should exit(1) when unknown options are given', () => {
    expect(run(['-f']).status).toBe(1);
  });
```

File: tests/init.test.js [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    it('should create correct tuture.yml with default values', () => {
+      expect(fs.existsSync('tuture.yml')).toBe(true);
+
+      const tuture = yaml.safeLoad(fs.readFileSync('tuture.yml'));
+      testTutureObject(tuture);
+    });
+  });
+});
+
+function testInit(args) {
```
Comment (pftom) [inline tests/init.test.js]: It seems  above two `describe` can use this `testInit` function to run test. 🌜

**Comment by pftom** [comment]
@mRcfps You guy write tests so clear, I enjoy reviewing it. 🙃

File: tests/tuture.test.js (hunk 1 above)
Comment (mRcfps) [reply tests/tuture.test.js]: OK, I'll add short option alias.

File: tests/init.test.js (hunk 2 above)
Comment (mRcfps) [reply tests/init.test.js]: Good catch! Actually I forgot to put `testInit` back to tests after debugging 🤯.

**Pull request merged**